File: 21-find-max-zero/21-find-max-zero.js
/*
    @param nums:{[Integer]}
    @return :{Integer}
*/
function findMaxZero(nums) {
  let count = 0;
  let max = 0;
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] === 0) {
      count += 1;
    } else {
      count = 0;
    }
    if (count > max) {
      max = count;
    }
  }
  return max;
}

console.log(findMaxZero([0, 1, 0, 0, 0, 1, 0, 1]));

File: 21-find-max-zero/21-find-max-zero.py
"""
    @param nums:{[Integer]}
    @return :{Integer}
"""


def findMaxZero(nums):
    zero_lst = ''.join(map(str, nums)).split('1')
    return max(len(item) for item in zero_lst)


print(findMaxZero([0, 0, 0]))
